product_docs/docs/biganimal/release/using_cluster/managing_replicas.mdx

```diff
@@ -30,12 +30,32 @@ You can create faraway replicas in any active regions in your cloud. There's no
    - `max_wal_senders`
    - `max_worker_processes`
 
-1. On the **Additional Settings** tab, in the **Backups** section, change the default replica backup retention period of 30 days using the **Retention Time** controls. You can configure the retention period as follows:
+1. On the **Additional Settings** tab, 
+
+   Under the **Backups** section, change the default replica backup retention period of 30 days using the **Retention Time** controls. You can configure the retention period as follows:
    - 1&ndash;180 days
 
    - 1&ndash;25 weeks
    - 1&ndash;6 months
 
+   Enable the **Custom Maintenance Window** option and use the controls to set a weekly 60-minute maintenance window in which maintenance upgrades occur for the cluster. If you don't set a window, the updates are applied at EDB's discretion with prior notification.  
+
+   !!! Note
+   Typically, maintenance updates take only a few minutes to complete.
+   !!!
+
+   For more information, see [Periodic maintenance](/biganimal/release/overview/updates).
+
+   Under the **Extensions** section, 
+   - enable the **pgvector** extension to add support for vector storage and vector similarity search to Postgres.
+   - enable the **PostGIS** extension to add support for storing, indexing and querying geographic data.
+
+   Under the **Security** section, enable the **Transparent Data Encryption (TDE)**. This option is available only when your primary cluster is TDE-enabled. It automatically enables TDE and allows you to select the encryption key from the available List.
```

Comment:
TDE should be enabled here by default for replicas of TDE-enabled clusters. cc @pramod-ahire

Reply:
Yes, it will be by default enabled & selected source cluster key for `Replica & Restore cluster`
<img width="1720" alt="Screenshot 2024-03-22 at 9 53 55 AM" src="https://github.com/EnterpriseDB/docs/assets/70689482/92287f71-5e8f-4122-92e3-6ba2d9093920">